Geometry/HcalCommonData/interface/HcalParametersFromDD.h

```diff
@@ -0,0 +1,17 @@
+#ifndef HcalCommonData_HcalParametersFromDD_h
+#define HcalCommonData_HcalParametersFromDD_h
+
+class DDCompactView;
+class HcalParameters;
+
+class HcalParametersFromDD {
+public:
+  HcalParametersFromDD() {}
+  virtual ~HcalParametersFromDD() {}
+
+  bool build(const DDCompactView*,  HcalParameters& );
+private:
+  static const int nEtaMax=100;
```

Comment:
@bsunanda - it is better to use constexpr and since it is private, move it to .cc